rcsbsearchapi/search.py

```diff
@@ -393,6 +402,115 @@ def __init__(self, structure_search_type: StructSimEntryType = "entry_id",
         super().__init__(service=STRUCT_SIM_SEARCH_SERVICE, params=parameters)
 
 
+class StructureMotifResidue():
+    """This class is for defining residues. Should only be used as a dictionary.
+    Defining Residues should be done in the following order:
+    Residue(label_asym_id, struct_oper_id, label_seq_id).
+
+    There are no default arguments. This MUST be done by the user in this order."""
+    def __init__(self, label_asym_id: str, struct_oper_id: str, label_seq_id, exchanges: Optional[list[StructMotifExchanges]] = None):
+        self.label_asym = label_asym_id
+        self.struct_oper_id = struct_oper_id
+        self.label_seq_id = label_seq_id
+
```

Comment:
Hmm, this looks OK for sure, but it differs a little from how StructureSimilarity is initialized, since that uses `chain_id` instead of `asym_id` (since "chain ID" is what appears on the front end and is what more general users are familiar with). Would changing that to be consistent with the StructureSimilarity be a lot of trouble? If so, then I guess we can leave it as is...